travis/script.ppc64el-clang.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-set -ev
-export QEMU_CPU=POWER8
-cd /build/build-cross
-make -j 2 all
```

Comment:
It was nice we could claim this on PPC. I understand you are removing it because it is very slow. Why is that? Would it be possible to move it to Jenkins?

Reply:
VSX testing is already made on Jenkins.
I don't know why VSX vector operations are slow on QEMU, but it is slower than AArch64 NEON emulation.